requirements.txt

```diff
@@ -1,13 +1,14 @@
 -r lint-requirements.txt
+huggingface_hub>=0.0.6
 hyperpyyaml>=0.0.1
 joblib>=0.14.1
 numpy>=1.17.0
 packaging
 pre-commit>=2.3.0
+sacrebleu>=1.5.1
```

Comment:
Please remove it from the general requirements.txt and instead try to see if it's installed when importing. If not, then print a message to install it ?